docs/machine-learning/how-to-guides/matchup-app-infer-net.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Create a game match up list app with Infer.NET and probalistic programming
+description: Discover how to use probalistic programming with Infer.NET to create a game matchup list app based on a simplified version of TrueSkill.
+ms.date: 10/04/2018
+ms.topic: how-to
+ms.custom: mvc
+#Customer intent: As a developer, I want to use probalistic programming with Infer.NET to create a game matchup list app based on a simplified version of TrueSkill.
+---
+# Create a game match up list app with Infer.NET and probabilistic programming
+
+This how-to guide teaches you about probabilistic programming using Infer.NET. This is an approach to machine learning where bespoke models are expressed as computer programs. It allows for incorporating domain knowledge in the models and makes the machine learning system more interpretable. It also supports online inference – the process of learning as new data arrives. Infer.NET is a framework for Infer.NET is used in various products at Microsoft in Azure, Xbox, and Bing.
```

Comment:
bespoke models -> unless this a well known term in ML, I'd pick a more common term for bespoke

Infer.NET is a framework for Infer.NET?

Infer.NET is used in various Microsoft products such as Azure, Xbox, and Bing.

This is an approach -> Probabilistic programming is an approach?